pkg/vm/vm.go

```diff
@@ -345,7 +345,7 @@ func (vm *VirtualMachine) computeRes(
 	case AddOperands:
 		op0 := op0Cell.Read()
 		op1 := op1Cell.Read()
-		return mem.EmptyMemoryValueAs(op0.IsAddress()).Add(op0, op1)
+		return mem.EmptyMemoryValueAs(op0.IsAddress() || op1.IsAddress()).Add(op0, op1)
```

Comment:
Review meeting ; suggested by Rodrigo and resolves prior problems (Felt + Addr addition error)